src/app/components/PageNotFound.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+/**
+* The page the user gets when an invalid route is entered
+* in the address bar.
+* @returns react elements.
+*/
+const pageNotFound = () => (
+  <div>
+    <h3 style={{ textAlign: 'center' }}>
+      404 Error... Testing me huh????
+    </h3>
```

Comment:
You can add a link or button to take the user back to the homepage or news sources page.